core/domain/email_manager.py

```diff
@@ -351,6 +351,117 @@
         'Review')
 }
 
+NOTIFICATION_FOR_CONTRIBUTOR_RANKING_ACHIEVEMENT: Dict[str, Dict[str, str]] = {
+    feconf.CONTRIBUTION_TYPE_TRANSLATION: {
+        feconf.CONTRIBUTION_SUB_TYPE_ACCEPTANCE: {
+            'email_body_template': (
+                'Hi %s,<br><br>'
+                'This is to let you know that you have successfully achieved '
+                'the %s rank for submitting translations in %s. Your efforts '
+                'help Oppia grow better every day and support students around '
+                'the world.<br><br>'
+                'You can check all the achievements you earned in the '
+                '<a href="%s%s">Contributor Dashboard</a>.<br><br>'
+                'Best wishes and we hope you can continue to contribute!'
+                '<br><br>'
+                'Best wishes,<br>The Oppia Contributor Dashboard Team'
```

Comment:
```suggestion
                'The Oppia Contributor Dashboard Team'
```

Seems redundant with 'Best wishes` above. Ditto below.

Reply:
Done